src/styles/_clock.scss

```diff
@@ -0,0 +1,60 @@
+.clock {
+  @include size(600px, 600px);
```

Comment:
Import used SCSS files explicit. Always prefer explicit over implicit. It makes code more understandable and helps to avoid many bugs. Also, with import IDE provide better autocomplete for you.